xbmc/network/upnp/UPnPInternal.cpp

```diff
@@ -141,15 +147,34 @@ GetMimeType(const CFileItem& item,
 
     NPT_String mime;
 
-    /* We always use Platinum mime type first
-       as it is defined to map extension to DLNA compliant mime type
-       or custom according to context (who asked for it) */
-    if (!ext.IsEmpty()) {
+    if (!ext.IsEmpty())
+    {
+      /* We look first to our extensions/overrides of libplatinum mimetypes. If not found, fallback to
+         Platinum definitions.
+      */
+      auto KodiOverrideMimeType = std::find_if(
```

Comment:
Can be `const`. And shouldn't all local variables start with a lower case letter?